CMSgov/design-system
# [NO-TICKET] Loki.sh - script to automate test running with loki/storybook

## Summary

Makes it easier to run Loki visual regression tests!

_Doesn't check if docker is currently running, I suppose it could? Not sure how that would work on Jankins.. Whatcha think?_

### Added

Loki.sh script which takes a single argument, if none passed it defaults to working on core, otherwise `healthcare` and `medicare` will run their respective yarn jobs

## How to test

Have docker running.. 

Possible `args` are: `update` `test` `approve`

`./loki.sh args` `./loki.sh -t healthcare args` `./loki.sh -t medicare args` .. See that it runs the tests and kills the storybook process when complete regardless of success/failure.
